todo/tests/integration/test_tasks_delete.py

```
@@ -1,42 +1,43 @@
+        cls.db.tasks.insert_one(task_doc)
+        cls.existing_task_id = str(task_doc["_id"])
+        cls.non_existent_id = str(ObjectId())
+        cls.invalid_task_id = "invalid-task-id"
+        cls.client = APIClient()
 
-    @patch("todo.repositories.task_repository.TaskRepository.delete_by_id")
-    def test_delete_task_success(self, mock_delete_by_id):
-        url = reverse("task_detail", args=[self.task_id])
-        response = self.client.delete(url)
-        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+    def test_delete_task_success(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)